app/helpers/application_helper/button/vm_instance_scan.rb

```diff
@@ -1,6 +1,17 @@
 class ApplicationHelper::Button::VmInstanceScan < ApplicationHelper::Button::Basic
   needs_record
 
+  def calculate_properties
+    super
+    if disabled?
```

Comment:
This does not match the method name below.  Prefer `disabled?` if the question is "is this disabled?".  Prefer `disable?` if the question is "should this be disabled?"